tests/modules/keyspace_events.c

```diff
@@ -40,6 +40,8 @@
 RedisModuleDict *loaded_event_log = NULL;
 /** stores all the keys on which we got 'module' keyspace notification **/
 RedisModuleDict *module_event_log = NULL;
+/** stores all the keys on which we got 'unlink' keyspace notification **/
+RedisModuleDict *unlink_event_log = NULL;
```

Comment:
if you agree with the "removed" term, please update all the remaining references.